src/arrayMethodSort.js

```diff
@@ -4,9 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const previousValue = this[i - 1];
+        const currentValue = this[i];
+
+        if (compareFunction(previousValue, currentValue) > 0) {
+          this[i - 1] = currentValue;
+          this[i] = previousValue;
+          count++;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+function compareAsStrings(a, b) {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  return stringA > stringB
+    ? 1
+    : stringA < stringB
+      ? -1
+      : 0;
```

Comment:
Using a ternary operator in the ternary operator is bad practice. Use if/else instead